lib/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "opcua"
-version = "0.12.0" # OPCUARustVersion
+version = "0.12.0"                               # OPCUARustVersion
 description = "OPC UA client and server API"
 authors = ["Adam Lock <locka99@gmail.com>"]
 homepage = "https://github.com/locka99/opcua"
 repository = "https://github.com/locka99/opcua"
 license = "MPL-2.0"
-keywords = ["opcua","opc","ua"]
-categories = ["embedded","network-programming"]
+keywords = ["opcua", "opc", "ua"]
+categories = ["embedded", "network-programming"]
```

Comment:
Maybe we could leave this change out of this PR and instead aim for something like toml-formatting?

Reply:
yes, it was just the "autoformat on save"